EnglishFilter/sections/whitelist.txt

```diff
@@ -1969,9 +1969,6 @@ pornhub.com,pornhub.org,pornhub.net,pornhubthbh7ap3u.onion#@#[style] > div > ifr
 pornhub.com,pornhub.org,pornhub.net,pornhubthbh7ap3u.onion#@#.sectionWrapper > div[class*=" "]:not([class="sectionTitle"]):not([class="reset"])
 ! https://github.com/AdguardTeam/AdguardForWindows/issues/1273
 @@||southpark.cc.com^$jsinject
-! https://github.com/AdguardTeam/AdguardForWindows/issues/2063
-! https://github.com/AdguardTeam/AdguardFilters/issues/8669
-@@||sunbtc.space^$jsinject
```

Comment:
Checked with AG for Windows?

Reply:
I checked it again with nightly and it seems that the old issue is still here.
@Sergey-Lyapin plz restore this exception, but limit it to the platform for apps (for `windows` this is definitely necessary).
Also check other apps.

Reply:
windows, macos, android - all affected(corelibs issue)

Reply:
For me: windows, macos - problem is reproducing, android - is not reproducing

Reply:
ok